Rick-and-Morty/Rick And Morty/Factories/CharacterCardStateFactory.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CharacterCardStateFactory.swift
+//  Rick And Morty
+//
+//  Created by Scottie Gray on 2021-08-13.
+//  Copyright © 2021 Novoda. All rights reserved.
+//
+
+import Foundation
+
+final class CharacterCardStateFactory {
+    private func getIsAlive(character: Character) -> Bool {
+        if character.status == "Alive" {
+            return true
+        }
+        
+        return false
+    }
```

Comment:
If I remember correctly there is also state `unknown`. is the bool correct choice here?